server/models/User.coffee

```diff
@@ -89,7 +90,10 @@ UserSchema.methods.isStudent = ->
 UserSchema.methods.getUserInfo = ->
   id: @get('_id')
   email: if @get('anonymous') then 'Unregistered User' else @get('email')
-  
+
+UserSchema.methods.getYearSubscriptionGroup = ->
+  core_utils.getYearSubscriptionGroup(@get('testGroupNumber'))
```

Comment:
Yea, a little whacky but this our best solution at the moment.  We already do this in other places, otherwise I'd spend more time figuring out a better solution than introducing this pattern.